test/Base.t.sol

```diff
@@ -84,7 +93,7 @@ abstract contract Base_Test is Assertions, Calculations, DeployOptimized, Modifi
         nftDescriptorMock = new NFTDescriptorMock();
 
         // Set the Sablier fee on the Merkle factory.
-        merkleFactory.setDefaultSablierFee(defaults.DEFAULT_SABLIER_FEE());
+        merkleFactory.setDefaultSablierFee(SABLIER_FEE);
```

Comment:
Lets revert it since this is going to be removed in https://github.com/sablier-labs/v2-core/pull/1089